frontend/jest.config.ts

```diff
@@ -9,6 +9,7 @@ const config: Config.InitialOptions = {
 	modulePathIgnorePatterns: ['dist'],
 	moduleNameMapper: {
 		'\\.(css|less|scss)$': '<rootDir>/__mocks__/cssMock.ts',
+		'\\.md$': '<rootDir>/__mocks__/cssMock.ts',
```

Comment:
NIT: rename the mock file from `cssMock.ts`

Reply:
they should be referring to same file content, hence didn't created a duplicate, will do in next PR

Reply:
yes so update it to a generic name. not super important but better to maintain code quality